synapse/lib/storm.py

```diff
@@ -924,15 +924,29 @@
             ('--form', {'help': 'Form to fire on.'}),
             ('--tag', {'help': 'Tag to fire on.'}),
             ('--prop', {'help': 'Property to fire on.'}),
-            ('--query', {'help': 'Query for the trigger to execute.', 'required': True}),
+            ('--query', {'help': 'Query for the trigger to execute.', 'required': True,
+                         'dest': 'storm', }),
             ('--async', {'default': False, 'action': 'store_true',
-                            'help': 'Make the trigger run in the background.'}),
+                         'help': 'Make the trigger run in the background.'}),
             ('--disabled', {'default': False, 'action': 'store_true',
                             'help': 'Create the trigger in disabled state.'}),
             ('--name', {'help': 'Human friendly name of the trigger.'}),
         ),
         'storm': '''
-            $trig = $lib.trigger.add($cmdopts)
+            $opts = $lib.copy($cmdopts)
```

Comment:
Why does this create a copy? Each execution of a command gets it's own `$cmdopts` dict that it is free to modify :)

Reply:
cmdopts structures are not mutable.

Reply:
👍